### PR TITLE
LOC CHECKIN | Microsoft/vstest master | 20190828

### PR DESCRIPTION
**Pull request by v-zbsail**
## Description
Please add a meaningful description for this change.
Ensure the PR has required unit tests.

Loc update

## Related issue
Kindly link any related issues. E.g. Fixes #xyz.

### CODE REVIEW

File: src/Microsoft.TestPlatform.CrossPlatEngine/Resources/xlf/Resources.pt-BR.xlf [hunk 1]
```diff
@@ -155,7 +155,8 @@
       <trans-unit id="TestHostExitedWithError">
         <source>Testhost process exited with error: {0}. Please check the diagnostic logs for more information.</source>
         <target state="translated">O processo do testhost foi encerrado com o erro: {0}. Verifique os logs de diagnóstico para obter mais informações.</target>
-        <note></note>
+        <note>
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.CrossPlatEngine/Resources/xlf/Resources.pt-BR.xlf]: @v-zbsail Why does this show up as a change ? We should instead change this to <note/> instead. ?

Comment (josefhosek) [reply src/Microsoft.TestPlatform.CrossPlatEngine/Resources/xlf/Resources.pt-BR.xlf]: Hi @singhsarab, this behavior is by design done in our building tool (adding new line between <note> tags). If it causes problems on your side, we can create post-process on our side to remove these.

Comment (singhsarab) [reply src/Microsoft.TestPlatform.CrossPlatEngine/Resources/xlf/Resources.pt-BR.xlf]: Yeah, please do, the xlf tool creates those for us, and we don't want to keep going back and forth on this.

Comment (josefhosek) [reply src/Microsoft.TestPlatform.CrossPlatEngine/Resources/xlf/Resources.pt-BR.xlf]: I'll keep a note on this and make sure to have those correct next time! Thanks

**Review by singhsarab — APPROVED** [review]

**Pull request merged**